### PR TITLE
git reset does not affect commit, but changes index, stage to them

### PR DESCRIPTION
**Pull request by randomecho**
Clear up what happens when you run `git reset` in that it does not affect the commit, or allow to change it per se, but that it affects what is sitting in the index and staging as pointed to by the commit or HEAD pointer mentioned.

This should make the notes actual now.

### CODE REVIEW

**Comment by matthewmccullough** [comment]
Looks like it nicely addresses the concerns in #63.  Will want other @github/training-team members to check.

File: basic/index.html [hunk 1]
```diff
@@ -691,7 +691,7 @@
 
     <h4>
       git reset --soft
-      <small>undo the last commit</small>
+      <small>moves what HEAD points to, index and staging are untouched</small>
```
Comment (jordanmccullough) [inline basic/index.html]: Just a small modification to the description here?

"moves HEAD to specified commit reference, index and staging are untouched"

File: basic/index.html [hunk 2]
```diff
@@ -752,7 +753,8 @@
 
     <p>In the above example, while we had both changes ready to commit and
        ready to stage, a <code>git reset --hard</code> wiped them out.
-       On top of that, the last commit has been undone.</p>
+       The working tree and stage are now reset to the point just after the
+       commit pointed at by HEAD.</p>
```
Comment (jordanmccullough) [inline basic/index.html]: Thoughts on slight change to the verbiage here?

"The working tree and staging area are reset to the tip of the current branch or HEAD."

**Comment by randomecho** [comment]
Thanks for the revision notes. Totally on board with the edits.

**Comment by adailey14** [comment]
Hi everyone, what's the story with this pull request? Was about to recommend this site as a great way to learn git basics but this change has to be made first. Happy to do any work needed to make this happen.

**Comment by randomecho** [comment]
This looks merged.

**Comment by matthewmccullough** [comment]
@randomecho @adailey14 Thank you both! Merged and shipped!

**Pull request closed without merging**